docs/data-sources/equinix_fabric_connection.md

```diff
@@ -62,15 +63,16 @@ Read-Only:
 
 - `account` (Set of Object) (see [below for nested schema](#nestedobjatt--a_side--access_point--account))
 - `authentication_key` (String)
-- **Deprecated** `gateway` Use `router` instead (Set of Object) (see [below for nested schema](#nestedobjatt--a_side--access_point--router))
-- `router` CloudRouter; Replaces `gateway` attribute (Set of Object) (see [below for nested schema](#nestedobjatt--a_side--access_point--router))
+- `gateway` (Set of Object) (see [below for nested schema](#nestedobjatt--a_side--access_point--gateway))
```

Comment:
Curious how we lost the deprecation on this field

Reply:
Ah yes, I had fixed that in the resource, but forgot the data_source. Will update. Good comment.